src/main/resources/liquibase-maven-plugin.properties

```diff
@@ -0,0 +1,26 @@
+changeLogFile=classpath:liquibase-changeLog.xml
```

Comment:
You don't need this file. Just add `spring.liquibase.enabled=true` to your application.properties